### PR TITLE
Activities class weekly schedule 186900934

### PR DESCRIPTION
**Pull request by guiburato**
[#186900934](https://www.pivotaltracker.com/story/show/186900934)

- new screen activities-class-weekly-schedule .

### CODE REVIEW

File: src/main/java/org/ilgcc/app/inputs/Gcc.java [hunk 1]
```diff
@@ -18,4 +20,5 @@ public class Gcc extends FlowInputs {
 
     private String languageRead;
     private String languageSpeak;
+    private ArrayList<String> weeklySchedule;
```
Comment (sree-cfa) [inline src/main/java/org/ilgcc/app/inputs/Gcc.java:23]:
```suggestion
    private List<String> weeklySchedule;
```

**Review by sree-cfa — CHANGES_REQUESTED** [review]
Can you update the journey test?

**Comment by sree-cfa** [comment]
Missing **[Program Name]** per Figma 
Screenshot 2024-02-12 at 2 42 06 PM

File: src/main/java/org/ilgcc/app/inputs/Gcc.java (hunk 1 above)
Comment (guiburato) [reply src/main/java/org/ilgcc/app/inputs/Gcc.java]: Ops.. Good call.

**Comment by guiburato** [comment]
Screenshot 2024-02-13 at 3 39 29 PM

**Review by sree-cfa — APPROVED** [review]
lgtm

**Pull request merged**